src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderTemporaryAbsenceApplication.kt

```diff
@@ -38,18 +38,20 @@ interface OffenderTemporaryAbsenceApplicationDsl {
     transportType: String = "VAN",
     returnDate: LocalDate = LocalDate.now().plusDays(1),
     returnTime: LocalDateTime = LocalDateTime.now().plusHours(1),
+    toAddress: Address? = null,
     dsl: OffenderScheduledTemporaryAbsenceDsl.() -> Unit = {},
   ): OffenderScheduledTemporaryAbsence
 
   @OffenderTemporaryAbsenceApplicationMovementsDslMarker
-  fun movement(
+  fun outsideMovement(
```

Comment:
I think this is better language - there are lots of different kinds of "movement". This kind is a list of places the prisoner will visit while they're out on a TAP.